memoized_lcsh.txt

```diff
@@ -1 +1 @@
-{"Sailboats"=>"http://id.loc.gov/authorities/subjects/sh85116520", "Periodical cicada"=>"http://id.loc.gov/authorities/subjects/sh85099887", "Saguenay River (Quebec)"=>"https://id.loc.gov/authorities/subjects/sh86006143", "Social stratification"=>"http://id.loc.gov/authorities/subjects/sh99000315", "Originality"=>"http://id.loc.gov/authorities/subjects/sh85095646", "Sailing"=>"http://id.loc.gov/authorities/subjects/sh85116532", "Marsden"=>"https://id.loc.gov/authorities/subjects/sh99014562", " Sam (Fictitious character)"=>"https://id.loc.gov/authorities/subjects/sh2008000533", "Molds (Fungi)"=>"http://id.loc.gov/authorities/subjects/sh85086566", "Lindens"=>"http://id.loc.gov/authorities/subjects/sh85077148", "Statues"=>"http://id.loc.gov/authorities/subjects/sh2001008682", "Test family"=>"http://id.loc.gov/authorities/subjects/sh92004443", "Semantic Web"=>"https://id.loc.gov/authorities/subjects/sh2002000569", "Cancer"=>"http://id.loc.gov/authorities/subjects/sh85019492"}
\ No newline at end of file
+{"Semantic Web"=>"http://id.loc.gov/authorities/subjects/sh2002000569", "Abdomen--Cancer"=>"http://id.loc.gov/authorities/subjects/sh85000095"}
```

Comment:
I guess LCSH used the URI as an identifier and is stuck with the http scheme until they update it and add another URI.